src/module.c

```diff
@@ -830,6 +830,53 @@ int64_t commandKeySpecsFlagsFromString(const char *s) {
     return flags;
 }
 
+RedisModuleCommandProxy *moduleCreateCommandProxy(RedisModuleCtx *ctx, const char *name, RedisModuleCmdFunc cmdfunc, int64_t flags, int firstkey, int lastkey, int keystep) {
```

Comment:
i think you should re-order the commands and put this below RM_CreateCommand.
you'll need to add a forward declaration, but ti'll prevent a blame log change of so many lines...